packages/crowdstrike/data_stream/falcon/elasticsearch/ingest_pipeline/default.yml

```
@@ -78,6 +78,25 @@ processors:
                     }
                 } 
             }
+  # Handle event Tags, which can be a string or a list of maps.
+  - script:
+      tag: convert-Tags-and-copy-to-tags
+      description: Convert tags from nested type and append to ctx.tags.
+      lang: painless
+      if : ctx.crowdstrike?.event != null && ctx.crowdstrike.event.Tags != null
```

```suggestion
      if : ctx.crowdstrike?.event?.Tags != null
```

I think we can consolidate into 1 expression.